shared/components/JobsSection/index.scss

```diff
@@ -78,6 +78,9 @@
     &:first-child {
       padding-top: 20px;
     }
+    &__title {
+      margin-top: 10px;
```

Comment:
is there is already padding on the parent element of `__title` -  can we add 10px to this instead? It gets annoying if spacing that visually affects the same space is actually set in two different places.